internal/app/http/common.go

```diff
@@ -363,6 +363,16 @@ func toImportThingModelResponse(res repos.ImportResult) server.ImportThingModelR
 	}
 }
 
+func toInfoResponse() server.InfoResponse {
```

Comment:
this function name does not fit. It implies some kind of conversion of inputs, but the function has no arguments

Reply:
You are right. I renamed it to infoResponse().